test/publish/test_orcid.py

```diff
@@ -120,6 +141,8 @@ def test_export_author_orcids(
         assert rows[0].primary_school == "School of Engineering"
         assert rows[0].primary_department == "Computer Science"
         assert rows[0].primary_division == "Philanthropy Division"
+        assert rows[1].sunetid == "senaj"
+        assert rows[1].orcidid is None
```

Comment:
happy to flesh this out and make the check of the second user more thorough.  i just concentrated on whether we were looking at the right record, and whether it had the expected value for the field whose behavior is modified by this PR.